src/main/java/de/tum/in/www1/artemis/config/SecurityConfiguration.java

```diff
@@ -140,35 +140,36 @@ protected void configure(HttpSecurity http) throws Exception {
             .csrf()
             .disable()
             .addFilterBefore(corsFilter, UsernamePasswordAuthenticationFilter.class)
-                .exceptionHandling().authenticationEntryPoint(problemSupport)
-                .accessDeniedHandler(problemSupport)
-        .and()
+            .exceptionHandling().authenticationEntryPoint(problemSupport)
```

Comment:
please revert all changes in this file

Reply:
Sure, I reverted the auto-formatting changes.